pkg/verifier/aws/entry_point.go

```diff
@@ -84,6 +101,14 @@ func (a *AwsVerifier) ValidateEgress(vei verifier.ValidateEgressInput) *output.O
 		a.Output.AddError(err)
 	}
 
+	if cleanupSecurityGroup {
+		if _, err := a.AwsClient.DeleteSecurityGroup(vei.Ctx, &ec2.DeleteSecurityGroupInput{
+			GroupId: awsTools.String(vei.AWS.SecurityGroupId),
+		}); err != nil {
+			a.Output.AddError(err)
+		}
```

Comment:
```suggestion
			_, err := a.AwsClient.DeleteSecurityGroup(vei.Ctx, &ec2.DeleteSecurityGroupInput{GroupId: awsTools.String(vei.AWS.SecurityGroupId)})
		if err != nil {
			a.Output.AddError(err)
		}
```

Reply:
I think it would read cleaner here with the if before the err statement instead of the before the return parameters.